aws/hll/aws-s3-transfer-manager/src/error.rs

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_smithy_types::byte_stream;
+use std::io;
+
+// TODO(design): revisit errors
```

Comment:
Whomever does look at this should read https://smithy-lang.github.io/smithy-rs/design/rfcs/rfc0022_error_context_and_compatibility.html